src/renderer/scss/component/_shapeshift.scss

```diff
@@ -0,0 +1,63 @@
+.shapeshift__wrapper.shapeshift__initial-wrapper {
+  min-height: 311px;
+}
+
+.shapeshift__content {
+  .spinner {
+    margin-top: 75px;
+  }
+}
+
+.shapeshift__success {
+  color: var(--color-primary);
+}
+
+.shapeshift__error {
+  color: var(--color-error);
+}
+
+.shapeshift__error,
+.shapeshift__maxdeposit {
+  min-height: 25px;
+}
+
+.shapeshift__deposit-address-wrapper {
+  display: flex;
+  flex-direction: row;
+
+  * {
+    align-self: center;
+  }
+}
+
+.shapeshift__deposit-address {
+  max-width: 500px;
+  font-size: 22px;
```

Comment:
Does this actually require it's own styling? Is there no other styling that already exists that this should match?